### PR TITLE
fix: move instance type as inline argument

### PR DESCRIPTION
**Pull request by wtrocki**
Deprecation of the predefined parameters is not supported of our tooling. Additionally this parameter is defined only once and it should not be reused by kafka instance sizes feature branch etc so better to define it explicitly.

### CODE REVIEW

**Comment by wtrocki** [comment]
CC @JameelB @vmanley

**Comment by JameelB** [comment]
Thanks for the fix @wtrocki! do you mind regenerating the bindata when you get a chance? `make openapi/generate`

File: jameel.diff [hunk 1]
```diff
@@ -0,0 +1,80 @@
+diff --git a/openapi/kas-fleet-manager.yaml b/openapi/kas-fleet-manager.yaml
```
[the remark this answers is not in the document]
Comment (wtrocki) [reply jameel.diff:1]: Oh.. that was.. from before you know :) sorry :D

**Comment by vmanley** [comment]
lgtm

**Review by JameelB — APPROVED** [review]

**Review by ziccardi — APPROVED** [review]

**Comment by JameelB** [comment]
Not part of this PR but as a follow up question :) @wtrocki does this mean that if we deprecate a reusable parameter, we need to move it inline? Do you know if this is also the same case for any reusable components? If it is, it might be good to document this :thinking: Same as removing deprecated fields from 'required' (https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/898)

**Pull request merged**